src/collapse/sample.rs

```diff
@@ -70,26 +85,34 @@ impl Collapse for Folder {
             }
         }
 
+        // Process the data...
+        let mut occurrences = Occurrences::new(1);
         loop {
             line.clear();
-
             if reader.read_line(&mut line)? == 0 {
                 warn!("File ended before end of call graph");
-                return self.write_stack(&mut writer);
```

Comment:
I think this call is still needed, no?

Reply:
Yes - Will add it back. Should we make this an error? I'll leave it as a warning for now, but let me know if you think it should be cause us to return an error instead.

Reply:
Yes, I would make this an error!